stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/DOM.java

```diff
@@ -88,12 +91,18 @@ public void highlightNode(JsonRpcPeer peer, JSONObject params) {
     if (request.nodeId == null) {
       LogUtil.w("DOM.highlightNode was not given a nodeId; JS objectId is not supported");
     } else {
-      RGBAColor contentColor = request.highlightConfig.contentColor;
+      final RGBAColor contentColor = request.highlightConfig.contentColor;
       if (contentColor == null) {
         LogUtil.w("DOM.highlightNode was not given a color to highlight with");
       } else {
-        Object element = mObjectIdMapper.getObjectForId(request.nodeId);
-        mDOMProvider.highlightElement(element, contentColor.getColor());
+        final Object element = mObjectIdMapper.getObjectForId(request.nodeId);
+
+        mDOMProvider.postAndWait(new Runnable() {
```

Comment:
This strictly adds delay per our discussion.  Prefer `highlightElement` be made safe to run on any arbitrary thread to avoid this perf hit.